src/GrumPHP/Console/Command/Git/PrePushCommand.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace GrumPHP\Console\Command\Git;
+
+use GrumPHP\Collection\FilesCollection;
+use GrumPHP\Configuration\GrumPHP;
+use GrumPHP\Console\Helper\PathsHelper;
+use GrumPHP\Console\Helper\TaskRunnerHelper;
+use GrumPHP\IO\ConsoleIO;
+use GrumPHP\Locator\ChangedFiles;
+use GrumPHP\Runner\TaskRunnerContext;
+use GrumPHP\Task\Context\GitPrePushContext;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * This command runs the git pre-commit hook.
+ */
+class PrePushCommand extends Command
+{
+    const COMMAND_NAME = 'git:pre-push';
+
+    /**
+     * @var GrumPHP
+     */
+    protected $grumPHP;
+
+    /**
+     * @var ChangedFiles
+     */
+    protected $changedFilesLocator;
+
+    /**
+     * @param GrumPHP $grumPHP
+     * @param ChangedFiles $changedFilesLocator
+     */
+    public function __construct(GrumPHP $grumPHP, ChangedFiles $changedFilesLocator)
+    {
+        parent::__construct();
+
+        $this->grumPHP = $grumPHP;
+        $this->changedFilesLocator = $changedFilesLocator;
+    }
+
+    /**
+     * Configure command
+     */
+    protected function configure()
+    {
+        $this->setName(self::COMMAND_NAME);
+        $this->addOption(
+            'skip-success-output',
```

Comment:
You won't need this option here. It was added since pre-commit runs before commit-msg. 
This means that you could see 2 ascii images during 1 commit.